Cargo.toml

```diff
@@ -7,7 +7,7 @@ version = "0.25.2"
 edition = "2021"
 
 [dependencies]
-peridio-sdk = { git = "https://github.com/peridio/peridio-rust.git", tag = "0.1.0" }
+peridio-sdk = { git = "https://github.com/peridio/peridio-rust.git", rev = "32c569e" }
```

Comment:
After the SDK gets merged tag it with 0.2.0 so that we can reference a tag here instead of a SHA.